src/routes.tsx

```diff
@@ -14,72 +14,90 @@ import {
   blockHeightParamLoader,
   transactionParamLoader,
   layerLoader,
+  networkLoader,
 } from './app/utils/route-utils'
 import { searchParamLoader } from './app/components/Search/search-utils'
 import { RoutingErrorPage } from './app/pages/RoutingErrorPage'
+import { NetworkThemeBubble } from './app/components/NetworkThemeBubble'
+import { useNetworkParam } from './app/hooks/useNetworkParam'
+import CssBaseline from '@mui/material/CssBaseline'
+
+const NetworkSpecificPart = () => (
+  <NetworkThemeBubble network={useNetworkParam()}>
+    <CssBaseline />
+    <Outlet />
+  </NetworkThemeBubble>
+)
 
 export const routes: RouteObject[] = [
   {
     errorElement: <RoutingErrorPage />,
     children: [
       {
         path: '/',
-        element: <HomePage />,
-      },
-      {
-        path: '/:network/search', // ?q=
-        element: <SearchResultsPage />,
-        loader: searchParamLoader,
+        element: <ThemedHomePage />,
       },
```

Comment:
OK, so it's undoubtedly true that the testnet/mainnet setting (of the home page) should be reflected in the URL, but I think it's more like a query parameter, and not part of the path, since we are not actually going anywhere, just planning to go... will do it separately.

Reply:
I think it is part of the path, and it should affect useGetStatus, globe colors, globe navigation
(and I would propose that is should affect search too)

Reply:
Okay, apparently it should only affect globe, and we are putting into query parameter now